Classes/Flow2Lab/EmberAdapter/Configuration/Source/YamlModelConfigurationSource.php

```diff
@@ -43,8 +43,7 @@ public function getPriority() {
 	 * @throws ConfigurationNotAvailableException
 	 */
 	public function getClassNameByObject($object) {
-		// todo: make configurable
-		throw new ConfigurationNotAvailableException();
+		return $this->reflectionService->getClassNameByObject($object);
```

Comment:
Just throw the ConfigurationNotAvailableException here to fallback to the reflection configuration source.